### PR TITLE
chore(doc): change the description of the project

### PR DESCRIPTION
**Pull request by nicolaferraro**
Wdyt?

### CODE REVIEW

**Comment by oscerd** [comment]
But lets discuss

**Comment by davsclaus** [comment]
Really like the tag-line. I posted it also on twitter.

I also think like Andrea we should link to its parent Apache Camel. And vice-versa, we should also from Apache Camel link to CamelK.

**Comment by drekbour** [comment]
FYI "Apache Camel K is lightweight integration platform" is not grammatically correct without that *a*!

**Comment by nicolaferraro** [comment]
> FYI "Apache Camel K is lightweight integration platform" is not grammatically correct without that _a_!

Oops! :+1:

File: README.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@ image:https://godoc.org/github.com/apache/camel-k?status.svg["GoDoc", link="http
 image:https://img.shields.io/github/license/openshift/origin.svg?maxAge=2592000["Licensed under Apache License version 2.0", link="https://www.apache.org/licenses/LICENSE-2.0"]
 image:https://badges.gitter.im/apache/camel-k.png["Chat on Gitter", link="https://gitter.im/apache/camel-k"]
 
-Apache Camel K (a.k.a. Kamel) is a lightweight integration framework built from Apache Camel that runs natively on Kubernetes and is specifically designed for serverless and microservice architectures.
+Apache Camel K is lightweight integration platform, born on Kubernetes, with serverless superpowers.
```
Comment (valdar) [inline README.adoc]: as pointed out: 

> ... is **a** lightweight ...

**Comment by nicolaferraro** [comment]
Fixed the typo. Don't know how we can enforce it belongs to Apache Camel. Do you have suggestions?

**Comment by davsclaus** [comment]
+1 LGTM

**Comment by nicolaferraro** [comment]
Well... let's do it!

**Pull request merged**